backends/cuda-gen/ceed-cuda-gen-operator-build.cpp

```diff
@@ -776,6 +776,9 @@ extern "C" int CeedCudaGenOperatorBuild(CeedOperator op) {
   code << devFunctions;
 
   string qFunction(qf_data->qFunctionSource);
+  string qFunctionName(qf_data->qFunctionName);
+  string oper;
+  oper = "kernel_" + qFunctionName;
```

Comment:
Should this be more namespaced? Seems like there could be lots of places within a production application that would have as much claim to `kernel_mass` as we do. Perhaps something like this?
```suggestion
  oper = "CeedKernel_Cuda_gen_" + qFunctionName;
```